playground/app.vue

```
@@ -1,49 +1,19 @@
-  // random operation to demonstrate updating the planet field programmatically
-  function updateVal(testUpdate: boolean) {
-    setValue('planet', (v) => {
-      const [base, count] = v.split('.')
-      const recoveredNum = Number(count ?? 0)
-      const safeNum = Number.isNaN(recoveredNum) ? 0 : recoveredNum
-      const BASE_DEFAULT = 'Jupiter'
-      return testUpdate ? `${base || BASE_DEFAULT}.${safeNum + 1}` : v
-    })
-  }
+  const schema = z.object({ fruit: z.string() })
+  const { register, getValue } = useForm({ schema, key: 'example-form' })
```

ESLint

[error] 'useForm' is not defined.

(no-undef)